centreon/www/front_src/src/Authentication/SAML/Form/inputs.ts

```diff
@@ -0,0 +1,336 @@
+import { not, prop } from 'ramda';
+import { FormikValues } from 'formik';
+
+import { InputType } from '@centreon/ui';
+import type { InputProps } from '@centreon/ui';
+
+import {
+  labelBothIdentityProviderAndCentreonUI,
+  labelCentreonUIOnly,
+  labelCertificate,
+  labelDefineRelationBetweenRolesAndAclAccessGroups,
+  labelEmailAttribute,
+  labelEnableSAMLAuthentication,
+  labelFullNameAttribute,
+  labelEntityIdURL,
+  labelLogoutFrom,
+  labelLogoutUrl,
+  labelRemoteLoginUrl,
+  labelSAMLOnly,
+  labelUserIdAttribute
+} from '../translatedLabels';
+import {
+  labelActivation,
+  labelAuthenticationConditions,
+  labelAutoImportUsers,
+  labelGroupsMapping,
+  labelIdentityProvider
+} from '../../translatedLabels';
+import {
+  labelAclAccessGroup,
+  labelApplyOnlyFirtsRole,
+  labelAuthenticationMode,
+  labelConditionValue,
+  labelConditionsAttributePath,
+  labelContactGroup,
+  labelContactTemplate,
+  labelDefineAuthorizedConditionsValues,
+  labelDefinedTheRelationBetweenGroupsAndContactGroups,
+  labelDeleteRelation,
+  labelEnableAutoImport,
+  labelEnableAutomaticManagement,
+  labelEnableConditionsOnIdentityProvider,
+  labelGroupValue,
+  labelGroupsAttributePath,
+  labelMixed,
+  labelRoleValue,
+  labelRolesAttributePath,
+  labelRolesMapping
+} from '../../shared/translatedLabels';
+import {
+  accessGroupsEndpoint,
+  contactGroupsEndpoint,
+  contactTemplatesEndpoint
+} from '../../api/endpoints';
+
+const isAutoImportDisabled = (values: FormikValues): boolean =>
+  not(prop('autoImport', values));
+
+const isAutoImportEnabled = (values: FormikValues): boolean =>
+  prop('autoImport', values);
+
+const authenticationConditions: Array<InputProps> = [
+  {
+    dataTestId: 'saml_authenticationConditions.isEnabled',
```

Comment:
Not the subject of this PR, but with the right structure, we probably should not need any test ids. 
forms should be pretty well structured to allow semantic testing with ease, using testing library.

Reply:
Sometimes, we want specific data-testid's